shift_detector/precalculations/SorensenDicePrecalculation.py

```diff
@@ -71,6 +71,9 @@ def process(self, store: Store) -> dict:
         df1a, df1b = random_split(df1, [0.95, 0.05], seed=11)           # Baseline for df1
         df2a, df2b = random_split(df2, [0.95, 0.05], seed=11)           # Baseline for df2
 
+        if df1a.empty or df1b.empty or df2a.empty or df2b.empty:
```

Comment:
Wouldn't it be enough to check for b-datasets 😄

Reply:
Jep